src/wp-admin/includes/class-wp-list-table.php

```diff
@@ -1503,6 +1503,78 @@ public function ajax_response() {
 		die( wp_json_encode( $response ) );
 	}
 
+	/**
+	 * Generates views links.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param array $link_data {
+	 *     An array of link data.
+	 *
+	 *     @type string $url     The link URL.
+	 *     @type string $label   The link label.
+	 *     @type bool   $current Optional. Whether this is the currently selected view.
+	 * }
+	 * @return array An array of link markup. Keys match the $link_data input array.
+	 */
+	protected function get_views_links( $link_data = array() ) {
+		if ( ! $link_data || ! is_array( $link_data ) ) {
+			_doing_it_wrong(
+				__METHOD__,
+				sprintf(
+					/* translators: %s: The $link_data argument. */
+					__( 'The <code>%s</code> argument must be a non-empty array.' ),
+					'$link_data'
+				),
+				'6.1.0'
+			);
+
+			return array( '' );
+		}
```

Comment:
```suggestion
		if ( ! is_array( $link_data ) ) {
			_doing_it_wrong(
				__METHOD__,
				sprintf(
					/* translators: %s: The $link_data argument. */
					__( 'The <code>%s</code> argument must be an array.' ),
					'$link_data'
				),
				'6.1.0'
			);

			return array( '' );
		}
```

My logic being that if a plugin conditionally adds links based on options, enabled modules, the type of user account, etc then reaching this point without any links may be perfectly reasonable.